test/test.py

```diff
@@ -0,0 +1,108 @@
+import os
+import pytest
+import docker
+import csv
+
+from reference_directory import ReferenceDirectory
+
+
+HOME_DIRECTORY = os.path.expanduser('~')
+
+GRAPE_DOCKERFILE = 'containers/snakemake/Dockerfile'
+GRAPE_IMAGE_TAG = 'genx_relatives:latest'
+
+REFERENCE_DIRECTORY = os.path.join(HOME_DIRECTORY, 'ref')
+CONTAINER_REFERENCE_DIRECTORY = '/media/ref'
```

Comment:
Why do we need 2 reference directories? `REFERENCE_DIRECTORY` looks like `/home/user/ref` while `CONTAINER_REFERENCE_DIRECTORY` is `/media/ref`. First path is used only to initialize `ReferenceDirectory` class instance which later checks for path existence and downloads all files to `CONTAINER_REFERENCE_DIRECTORY` anyway.

Reply:
For the sake of clarity I just want to split two conceptually different paths: reference directory of the worker machine and the reference directory inside the container.